src/shared/treeview/nodes/defaultRegionNode.ts

```diff
@@ -4,15 +4,15 @@
  */
 
 import { TreeItemCollapsibleState } from 'vscode'
-import { RegionInfo } from '../../shared/regions/regionInfo'
-import { AWSTreeNodeBase } from '../../shared/treeview/nodes/awsTreeNodeBase'
-import { toMap, updateInPlace } from '../../shared/utilities/collectionUtils'
-import { CloudFormationNode, DefaultCloudFormationNode } from './cloudFormationNodes'
-import { RegionNode } from './regionNode'
+import { CloudFormationNode, DefaultCloudFormationNode } from '../../../lambda/explorer/cloudFormationNodes'
```

Comment:
RegionNode can be considered general, but not DefaultRegionNode (you can see it uses Explorer-specific nodes here).

Consider:
* moving this back to the explorer location
* renaming to `AwsExplorerRegionNode`